css/style.css

```diff
@@ -0,0 +1,34 @@
+#main-doc {
+    display: ;
+}
+
+#navbar {
+    /* position: fixed; */
+    position: sticky;
+    top: 0px;
+}
+
+#navbar a {
+    display: block;
+    text-decoration: none;
+    /* padding: 10px 0; */
+}
+
+@media screen and (max-width:500px) {
+    #main-doc {
+        display: block;
+
+    }
+
+    #navbar {
+        /* position: fixed; */
```

Comment:
Please always remove commented lines of code.
This makes your work messy and difficult to review